### ISSUE TITLE
emirge_makedb.py: please add an argument-option for providing pre-downloaded SILVA-db

### ISSUE THREAD

**Issue by jvollme** [csmiller/EMIRGE#34]
Apparently, running `emirge_makedb.py` from behind a proxy-server can be problematic. 

This could be worked-around most easily if users would have the option to download the databases from SILVA manually and provide them to `emirge_makedb.py` per argument.

**Comment by Ingway** [comment]
I'm also having trouble to run emirge_makedb.py. I have the following log when I try to install . Something wrong with vsearch maybe?
http://textuploader.com/d1g3p

**Comment by epruesse** [comment]
@Ingway Yes, you need vsearch 2.0.0 or newer. The v1 series did not know how to deal with gzipped input data.
